modules/caddyhttp/reverseproxy/httptransport.go

```diff
@@ -493,6 +493,10 @@ type TLSConfig struct {
 	// When specified, TLS will automatically be configured on the transport.
 	// The value can be a list of any valid tcp port numbers, default empty.
 	ExceptPorts []string `json:"except_ports,omitempty"`
+
+	// The list of elliptic curves to support. Caddy's
+	// defaults are modern and secure.
```

Comment:
Actually, it reverts to Go's default, which are ~.

Reply:
Interesting, Go's defaults are "mid" (is that what ~ means)? What would you suggest instead?

Reply:
With ~ I meant to refer to the quote: modern and secure. 😆